pom.xml

```diff
@@ -248,6 +247,21 @@
             <artifactId>message-queue-producer</artifactId>
             <version>1.0.4.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.github.Financial-Times</groupId>
+            <artifactId>content-model</artifactId>
```

Comment:
I removed the content-model dependency and added the AccessLevel enum to this project.